doc/design/assumeutxo.md

```diff
@@ -51,12 +51,6 @@ The utility script
 
 ## Design notes
 
-- A new block index `nStatus` flag is introduced, `BLOCK_ASSUMED_VALID`, to mark block
```

Comment:
The next paragraph refers to this deleted comment ("using
  the nStatus flag mentioned above") and should be updated too.